tutorials/continuous-aggs-tutorial.md

```diff
@@ -220,38 +240,90 @@ max | 2016-01-31 23:59:59
-materialization_hypertable | _timescaledb_internal._materialized_hypertable_2
+SELECT remove_continuous_aggregate_policy('cagg_rides_view');
+
+SELECT add_continuous_aggregate_policy('cagg_rides_view', 
+  start_offset => interval '1 month',
+  end_offset   => INTERVAL '1 hour',
+  schedule_interval => INTERVAL '30 minutes');
+
+SELECT view_name, schedule_interval, 
+  config ->> 'start_offset' as start_offset,
+  config ->> 'end_offset' as end_offset,
+  date_trunc('second',next_start::timestamp) as next_start,
+  materialization_hypertable_name
+FROM timescaledb_information.continuous_aggregates ca
+  INNER JOIN timescaledb_information.jobs j 
+    ON ca.materialization_hypertable_name = j.hypertable_name
+  WHERE view_name = 'cagg_rides_view';
+
+-[ RECORD 1 ]-------------------+---------------------------
+view_name                       | cagg_rides_view
+schedule_interval               | 00:30:00
+start_offset                    | 1 mon
+end_offset                      | 01:00:00
+next_start                      | 2020-10-29 15:59:51
+materialization_hypertable_name | _materialized_hypertable_9
 ```
 
-You will find more details about the API in the [documentation][tsdb_doc].
+Notice that the `start_offset` is not set to `1 mon`.
+
+## Using `WITH NO DATA` when creating a Continuous Aggregate [](with-no-data)
+
+If you have a lot of historical data, we suggest creating the continuous aggregate
+using the `WITH NO DATA` parameter for the `CREATE MATERIALIZED VIEW` command. Doing 
+so will allow the continuous aggregate to be created instantly (you won't have to wait 
+for the data to be aggregated on creation!). Data will then begin to populate as the 
+continuous aggregate policy begins to run.
+
+**However**, only data newer than `start_offset` would begin to populate the continuous
+aggregate. If you have historical data that is older than the `start_offset` INTERVAL, 
+you need to manually refresh history up to the current `start_offset` to allow
+real-time queries to run efficiently.
+
+Using the example in **Step 2** above, if we had years worth of data, a better approach 
+to creating the continuous aggregate is shown in the SQL below which includes `WITH NO DATA`
+and specifically refreshes the history separate from the continuous aggregate refresh policy:
+
+```sql
+CREATE MATERIALIZED VIEW cagg_rides_view WITH
+  (timescaledb.continuous)
+AS
+SELECT vendor_id, time_bucket('1h', pickup_datetime) as day,
+  count(*) total_rides,
+  avg(fare_amount) avg_fare,
+  max(trip_distance) as max_trip_distance,
+  min(trip_distance) as min_trip_distance
+FROM rides
+GROUP BY vendor_id, time_bucket('1h', pickup_datetime)
+WITH NO DATA;
+
+SELECT add_continuous_aggregate_policy('cagg_rides_view', 
+  start_offset => INTERVAL '1 week',
+  end_offset   => INTERVAL '1 hour',
+  schedule_interval => INTERVAL '30 minutes');
+
+CALL refresh_continuous_aggregate('cagg_rides_view', NULL, localtimestamp - INTERVAL '1 week');
```

Comment:
Wouldn't it be better to swap the order?  do the `CALL` first, then add the policy?